_G leaks fix and french translation update

mrbuds opened the pull request:
https://github.com/Vysci/LFG-Bulletin-Board/commit/90e3435856fa33f543ff3d576e31866659542062 fixes potential taints and breaking other addons
for https://github.com/Vysci/LFG-Bulletin-Board/commit/15085f0264696d9a2b56132bbddaab12f510b565 i used names found on wowhead

Vysci commented on LFGBulletinBoard/Dungeons.lua:
```diff
@@ -168,31 +168,31 @@ function GBB.GetDungeonNames()
+			["MT"] = 	"Auchindoun: Tombes-mana",
+			["CRYPTS"] = "Auchindoun: Cryptes Auchenaï",
+			["SETH"] = 	"Auchindoun: Les salles des Sethekk",
+			["OHB"] = 	"Grottes du Temps: Contreforts de Hautebrande d'antan",
+			["MECH"] = 	"Donjon de la Tempête: Le Méchanar",
+			["BM"] = 	"Grottes du Temps: Le Noir Marécage",
+			["MGT"] = 	"Terrasse des Magistères",
+			["SH"] = 	"Citadelle des Flammes Infernales: Les Salles brisées",
+			["BOT"] = 	"Donjon de la Tempête: Botanica",
+			["SL"] = 	"Auchindoun: Labyrinthe des ombres",
+			["SV"] = 	"Réservoir de Glissecroc : Le Caveau de la vapeur",
+			["ARC"] = 	"Donjon de la Tempête: L'Arcatraz",
 			["KARA"] = 	"Karazhan",
-			["GL"] = 	"Gruul's Lair",
-			["MAG"] = 	"Hellfire Citadel: Magtheridon's Lair",
-			["SSC"] = 	"Coilfang Reservoir: Serpentshrine Cavern",
-			["EYE"] = 	"The Eye",
+			["GL"] = 	"Repaire de Gruul",
+			["MAG"] = 	"Citadelle des Flammes Infernales: Le repaire de Magtheridon",
+			["SSC"] = 	"Réservoir de Glissecroc : Caverne du sanctuaire du Serpent",
+			["EYE"] = 	"L'Œil du cyclone",
 			["ZA"] = 	"Zul-Aman",
-			["HYJAL"] = "The Battle For Mount Hyjal",
-			["BT"] = 	"Black Temple",
-			["SWP"] = 	"Sunwell Plateau",
+			["HYJAL"] = "Sommet d'Hyjal",
+			["BT"] = 	"Temple noir",
+			["SWP"] = 	"Plateau du Puits de soleil",
 			["ONY"] = 	"Repaire d'Onyxia (40)",
 			["MC"] = 	"Cœur du Magma (40)",
```
You are overwriting English names. You need to put all the french names under a new variable

mrbuds replied:
Sorry but i don't get it, this fix english names in the dungeonNamesLocales.frFR table, dungeonNamesLocales.deDE / esMX / ruRU doesn't use english names

Vysci replied:
oh you are right, my bad I didn't realize it was already in the french table

Vysci commented:
Can you pull the latest changes, which include heroic filtering and add French localization to the newly introduced variables?

mrbuds commented:
I have rebase the PR, fixed a new global you added and removed Tool.CopyPast, i saw it wasn't using correct backdrop template, then started fixing a few bugs it has, before i saw it wasn't use anywhere.

About the new localization variables, i don't play the game in french myself but i play on a french server, i wanted to make the addon work better on french servers but i don't have time & will to translate the whole UI, your last heroic changes should work correctly without an update in french localization.
